### DISCUSSION TITLE
How can i get the the traceId of log  from the HandlerException

### DISCUSSION THREAD

**Discussion by sam6255** [apache/skywalking#8368] [Q&A] [answered]
Hi Sir:
   i use the skywalking grpc logback plugin to collect the log message. but i find the log message from the exceptionHandler cant get the trace id.  
skywalking version 8.7.0
![image](https://user-images.githubusercontent.com/7156850/147816934-640475f3-74ee-469a-bdac-df6354092a00.png)

**Comment by wu-sheng** [comment, marked as answer]
It depends how you handler works. Once it is in another thread, it is your responsibility to make the tracing context propagated.
